docker/lnd/entrypoint.sh

```
@@ -0,0 +1,29 @@
+  useradd -M -u $USERID -g $GROUPID lnd
+  chown -R $USERID:$GROUPID /home/lnd
+fi
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
+  echo "$0: assuming arguments for lnd"
+
+  set -- lnd "$@"
+fi
+
+if [ "$1" = "lnd" ] || [ "$1" = "lncli" ]; then
+  echo "Running as lnd user: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)